README.md

```diff
@@ -8,3 +8,14 @@ It consists of:
 * [translation code](https://github.com/estree/formal/blob/master/src/index.js)
 * [generated self-contained JSON representations](https://github.com/estree/formal/tree/master/formal-data)
 * [generated TypeScript definitions](https://github.com/estree/formal/tree/master/formal-data/typescript) :warning: *auto-generated ES6 definition has minor issue; see [DefinitelyTyped version](https://github.com/borisyankov/DefinitelyTyped/blob/master/estree/estree.d.ts) for manually fixed version*
+
+## Dependencies
+* Node and npm
+* babel (`npm install -g babel-cli`)
```

Comment:
No, you don't need neither this nor following line. Dependencies are installed automatically and in a safe (local) way.